content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -1,8 +1,152 @@
+
+- `outs`: List of <abbr>output</abbr> entries for this `.dvc` file. Typically
+  there is only one (but several can be added manually).
+- `deps` (optional): List of <abbr>dependency</abbr> entries for this stage,
+  only present when `dvc import` and `dvc import-url` are used. Typically there
+  is only one (but several can be added manually).
+- `meta` (optional): Arbitrary metadata can be added manually with this field.
+  Any YAML contents is supported. `meta` contents are ignored by DVC, but they
+  can be meaningful for user processes that read `.dvc` files.
+
+An _output entry_ can consist of these fields:
+
+- `md5` (optional): Hash value for the output file
+- `path`: Path to the output in the <abbr>workspace</abbr>, relative to the
+  location of the `.dvc` file
+- `cache` (optional): Whether or not DVC should cache the output. `true` by
+  default
+
+A _dependency entry_ consists of a these possible fields:
+
+- `path`: Path to the dependency, relative to the `wdir` path (always present)
+- `md5` (optional): MD5 hash for the dependency (most
+  [stages](/doc/command-reference/run))
+- `etag` (optional): Strong ETag response header (only HTTP <abbr>external
+  dependencies</abbr> created with `dvc import-url`)
+- `repo` (optional): This entry is only for external dependencies created with
+  `dvc import`, and can contains the following fields:
+
+  - `url`: URL of Git repository with source DVC project
+  - `rev`: Only present when the `--rev` option of `dvc import` is used.
+    Specific commit hash, branch or tag name, etc. (a
+    [Git revision](https://git-scm.com/docs/revisions)) used to import the
+    dependency from.
+  - `rev_lock`: Git commit hash of the external <abbr>DVC repository</abbr> at
+    the time of importing or updating (with `dvc update`) the dependency.
+
+Note that comments can be added to `.dvc` files and `dvc.yaml` using the
+`# comment` syntax.
+
+> `meta` fields and `#` comments are preserved among executions of the
+> `dvc repro` and `dvc commit` commands, but not when a `.dvc` file is
+> overwritten by `dvc add`,`dvc import`, or `dvc import-url`.
+
+## dvc.yaml files
+
+When you add commands to a pipeline with `dvc run`, the `dvc.yaml` file is
+created or updated. Here's a simple example:
+
+```yaml
+stages:
+  features:
+    cmd: jupyter nbconvert --execute featurize.ipynb
+    deps:
+      - data/clean
+    params:
+      - levels.no
+    outs:
+      - features
+    metrics:
+      - performance.json
+  training:
+    cmd: python train.py
+    deps:
+      - train.py
+      - features
+    outs:
+      - model.pkl
+    plots:
+      - logs.csv:
+          x: epoch
+          x_label: Epoch
+    meta: 'For deployment'
+    # User metadata and comments are supported.
+```
+
+`dvc.yaml` files consists of a group of `stages` with names provided explicitly
+by the user with the `--name` (`-n`) option of `dvc run`.
+
+Each stage's contents are similar to individual [`dvc` files](#dvcfiles) but
```

Comment:
`.dvc`? 

link is broken

(how many other links are broken like this???)

No need to put any parallels - stage file vs .dvc. They are play very different role. Or you can put something like "It has similar to .dvc files fields"

Reply:
This paragraph was removed per a previous comment you had (but I forgot to push this last commit this morning). I cannot find any more instances of `dvc` file without . or this broken link anchor in docs so it was the only one.